storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -462,7 +463,7 @@ static bool datafile_copy_backup(const char *filepath, uint thread_n) {
   of the filters value. */
 
   if (check_if_skip_table(filepath)) {
-    msg_ts("[%02u] Skipping %s.\n", thread_n, filepath);
+    xb::info() << "Skipping " << filepath;
```

Comment:
Shouldn't we add the `.` charcter in the end?

Reply:
I am not sure on adding . for everyline. It is bit inconsistent here too. Sometimes it makes copying the filename difficult because it ends with "."

Also, it is better to add internally (automatically), if we see it necessary. (like we add \n")